index.js

```diff
@@ -49,18 +119,45 @@ function Car() {
         + Should return a string "Playing with x", x being the favorite toy.
 */
 
-function Baby() {
+function Baby(name, age, favoriteToy) {
+  Person.call(this, name, age, favoriteToy);
```

Comment:
You don't need the favoriteToy in the Person.call since the Person class doesn't have a favoriteToy attribute.